<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>718</width>
        <height>480</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{background-color: rgb(247, 247, 247); color:rgb(51,51,51); border: 1px solid  rgb(247, 247, 247);outline:0px;}
QListWidget::Item{background-color: rgb(247, 247, 247);}
QListWidget::Item:hover{background-color: rgb(247, 247, 247); }
QListWidget::item:selected{
	background-color: rgb(247, 247, 247);
	color:black; 
    border: 1px solid  rgb(247, 247, 247);
}
QListWidget::item:selected:!active{border: 1px solid  rgb(247, 247, 247); background-color: rgb(247, 247, 247); color:rgb(51,51,51); } </string>
      </property>
      <property name="batchSize">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>200</height>
       </size>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>702</width>
         <height>182</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_send">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>发送Txt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_sendImg">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>发送img</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_sendFile">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>发送File</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
